api/modules/zomato/views.py

```diff
@@ -0,0 +1,52 @@
+import requests
+from rest_framework.decorators import api_view
+import requests_cache
+from datetime import timedelta
+from rest_framework import status
+from rest_framework.response import Response
+from api.modules.zomato.constants import BASE_URL, header
+from api.modules.zomato.zomato_response import Zomato_Response as Food
+
+
+hour_difference = timedelta(hours=1)
+requests_cache.install_cache(expire_after=hour_difference)
+
+
+@api_view(['GET'])
+def get_restaurants_all(request, latitude, longitude):
+    response = []
+    """
+    Returns restaurant details forecast for given city using coordinates
+    :param request:
+    :param latitude:
+    :param longitude:
+    :return: 503 if Zomato api fails
+    :return: 200 successful
+    """
+
+    try:
+
+        lat, long = 'lat=' + latitude, 'lon=' + longitude
+        comp = BASE_URL+lat+'&'+long
```

Comment:
use `url = ZOMATO_API_URL.format(latitude, longitude)`
and `ZOMATO_API_URL = "https://developers.zomato.com/api/v2.1/geocode?lat={0}&lon={1}"`